lib/commands/context/helpers.js

```diff
@@ -15,31 +15,31 @@ import {toDetailsCacheKey, getWebviewDetails, WEBVIEWS_DETAILS_CACHE} from './ca
 export const CHROME_BROWSER_PACKAGE_ACTIVITY = /** @type {const} */ ({
   chrome: {
     pkg: 'com.android.chrome',
-    activity: 'com.google.android.apps.chrome.Main',
+    activity: 'com.google.android.apps.chrome.IntentDispatcher',
```

Comment:
not sure if it is a good idea to change records. Perhaps, it makes sense to just add new ones

Reply:
maybe it also makes sense to autodetect the activity for the given package name using `cmd package resolve-activity --brief <pkg_name>` (only works on Android 7+)?

Reply:
Maybe just try if https://github.com/appium/appium-uiautomator2-driver/pull/730 works for you

Reply:
👍 Sure, we will try it.

Reply:
you don't need to wait for a release. You can simply checkout my branch locally and then install the driver from the local filesystem using `--source=local` CLI argument.

Reply:
ok